monai/data/image_writer.py

```diff
@@ -373,11 +373,11 @@ class ITKWriter(ImageWriter):
     output_dtype: DtypeLike = None
     channel_dim: int | None
 
-    def __init__(self, output_dtype: DtypeLike = np.float32, affine_lps_to_ras: bool = True, **kwargs):
+    def __init__(self, output_dtype: DtypeLike = np.float32, affine_lps_to_ras: bool | None = None, **kwargs):
```

Comment:
We should warn that the default value has changed here (with deprecated decorator?).

Reply:
Hi @ericspod ,

I think should use this API: `deprecated_arg_default`:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/utils/deprecate_utils.py#L230

Thanks.

Reply:
sure, I've changed to the previous default